embulk-util-aws-credentials/src/main/java/org/embulk/util/aws/credentials/AwsCredentials.java

```diff
@@ -160,8 +161,13 @@ public void refresh() { }
                 };
             }
 
+        case "default":
```

Comment:
To keep backward compatibility with previous releases, we decided to not change the behavior of `default` clause.